app/src/main/java/com/github/se/project/ui/lesson/SelectedTutorDetails.kt

```diff
@@ -376,15 +382,36 @@ private fun DisplayRatingGrade(grade: Double, testTag: String) {
 }
 
 @Composable
-private fun DisplayProfilePicture(testTag: String) {
-  Surface(
-      modifier = Modifier.size(48.dp),
-      shape = MaterialTheme.shapes.medium,
-      color = MaterialTheme.colorScheme.primary.copy(alpha = 0.3f)) {
-        Icon(
-            imageVector = Icons.Default.Person,
-            contentDescription = null,
-            modifier = Modifier.padding(8.dp).testTag(testTag),
-            tint = MaterialTheme.colorScheme.primary)
-      }
+private fun DisplayProfilePicture(testTag: String, tutorProfile: Profile) {
```

Comment:
great function, we should discuss about what we will do for the profile picture as it is only a placeholder for the moment!